website/www/site/i18n/home/en.yaml

```diff
@@ -35,15 +35,15 @@
 - id: home-ctas-title
   translation: "Get started"
 - id: home-model-title1
-  translation: "How Does it work?"
+  translation: "How Does It Work?"
```

Comment:
I'd use sentence case ("How does it work?") instead of title case, but this is okay too... better than the previous inconsistent usage.

Reply:
ditto